pkg/operator/ceph/nfs/nfs.go

```diff
@@ -240,5 +241,23 @@ func validateGanesha(context *clusterd.Context, n cephv1.CephNFS) error {
 		return fmt.Errorf("at least one active server required")
 	}
 
+	// We cannot run an NFS server if no MDS is running
+	// We cannot wait for the pool to exist in addRADOSConfigFile()
+	// but this will hold the orchestration and does not help much even a CephFilesystems CRD is injected
+	filesystems, err := context.RookClientset.CephV1().CephFilesystems(n.Namespace).List(metav1.ListOptions{})
```

Comment:
Doesn't this only list the filesystems the user wants? This doesn't guarantee that MDSes are running right?

Reply:
1. The check for the pool existing below is really the one that will verify that the necessary pool exists. 
1. NFS can also be run on top of an object store as mentioned in [this line](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/nfs.yaml#L10), right? That means that checking for the pool to exist is the only check we can rely on.